omod/src/main/webapp/pages/markPatientDead.gsp

```diff
@@ -60,7 +82,7 @@ ${ui.includeFragment("coreapps", "patientHeader", [patient: patient])}
                 <span>${ui.message("aijar.markpatientdeceased.causeofdeath")} (${ui.message("emr.formValidation.messages.requiredField.label")})</span>
             </label>
             <select name="causeOfDeath" id="cause-of-death">
-                <option>Select Cause Of Death</option>
```

Comment:
@slubwama Set the option value to "" otherwise the value is set to "Select Cause of Death" checkout other Ref App forms